pages/_app.js

```diff
@@ -95,6 +95,15 @@ class OpenCollectiveFrontendApp extends App {
     super.componentDidCatch(error, errorInfo);
   }
 
+  componentDidMount() {
+    Router.events.on('routeChangeStart', url => {
```

Comment:
Maybe we should use `routeChangeComplete` so that we don't register the change if the route change fails

```suggestion
    Router.events.on('routeChangeComplete', url => {
```